lib/iris/_representation/cube_summary.py

```diff
@@ -364,6 +377,11 @@ def add_scalar_section(section_class, title, *args):
         add_scalar_section(
             ScalarCoordSection, "Scalar coordinates:", cube, scalar_coords
         )
+        add_scalar_section(
```

Comment:
I note that the order of all possible sections is now this ...
```
vector
  Dimension coordinates:
  Mesh coordinates:
  Auxiliary coordinates:
  Derived coordinates:
  Cell measures:
  Ancillary variables:
scalar
  Mesh:
  Scalar coordinates:
  Scalar ancillary variables:
  Scalar cell measures:
  Cell methods:
  Attributes:
```
So, I think there _is_ good reason for the location of the new scalar-ancils section : since cell-methods and attributes are rather different in "kind" from all the other scalar components, we want them at the end, after the rest.
However, that means that we now list cell-measures **_before_** ancillaries in the vector part, but **_after_** them in the scalar part.

I think the only sensible alternative would be to swap the order of the _vector_ sections.  However, that changes existing behaviour. 
**? did you consider this + what do you think ?**

Reply:
I added `Scalar ancillary variables:` before `Scalar cell measures:` for alphabetical reasons as I couldn't really think of a better way to order them. But as you point out we already decided on the order in the vector section so I'm happy to just follow that.